defaults/main.yml

```
@@ -1,20 +1,24 @@
 # TIaaS Configuration
 tiaas_galaxy_db_url: postgres
 tiaas_galaxy_idsecret: '{{ galaxy_config.galaxy.id_secret | default("USING THE DEFAULT IS NOT SECURE!") }}'
+tiaas_show_advertising: true
 
-# TIaaS GDPR
-retain_extra_time: 12
+# Ask for consent to retain contact info for extra time
+tiaas_retain_contact_require_consent: true
+# Additional months to retain contact info
+tiaas_retain_contact_extra_months: 12
```

This lets us be completely backwards compat.
```suggestion
tiaas_retain_contact_extra_months: "{{ retain_extra_time | default(12) }}"
```